frontend/public/components/app.jsx

```diff
@@ -337,6 +339,42 @@ const CaptureTelemetry = React.memo(function CaptureTelemetry() {
   return null;
 });
 
+const WarningPolicyAlert = React.memo(function WarningPolicyAlert() {
+  const { t } = useTranslation();
+  const toastContext = useToast();
+  const [warningPolicy] = useWarningPolicy();
+
+  React.useEffect(() => {
+    if (warningPolicy?.warning) {
+      const docURL = getDocumentationURL(documentationURLs.warningPolicy);
+
+      toastContext.addToast({
+        variant: AlertVariant.warning,
+        title: t('public~Warning Policy'),
+        content: t(`This {{kind}} {{ name }} violates policy {{warning}}`, {
```

Comment:
```suggestion
        content: t(`The {{kind}} {{ name }} violates policy {{warning}}`, {
```

just in case we aren't on the details page for that resource

Reply:
@spadgett Will update, thanks